symbols/arm9.yml

```diff
@@ -414,12 +414,28 @@ arm9:
         The language ID appears to be used to index some global tables.
         
         return: language ID
+    - name: PreprocessString
+      address:
+        NA: 0x20223F0
+        EU: 0x20225EC
+      description: |-
+        An enhanced sprintf, which recognizes certain tags and replaces them with appropiate game values.
+        This function can also be used to simply insert values passed within the preprocessor args
+
```

Comment:
Kind of annoying, but with YAML you need to have spaces at the start of empty lines to match the indentation of the paragraphs. Same thing with the other blank line.